javascript/src/main/components/BasicCourseSearch/CourseSearchCourseStartEndQtr.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from "react";
+import { Form, Button } from "react-bootstrap";
+
+const CourseSearchCourseStartEndQtr = ({ setCourseJSON, fetchJSON }) => {
+
+    const [startQuarter, setStartQuarter] = useState("20211");
+    const [endQuarter, setEndQuarter] = useState("20211");
+    const [subjectArea, setSubjectArea] = useState("CMPSC   ");
+    const [courseNumber, setCourseNumber] = useState("");
+    const [courseSuf, setCourseSuf] = useState("");
+
+    const handleSubmit = (event) => {
+        //console.log(event);
+        event.preventDefault();
+        fetchJSON(event, { startQuarter, endQuarter, subjectArea, courseNumber, courseSuf}).then((courseJSON) => {
+            setCourseJSON(courseJSON);
+        });
+    };
+
+    // FALL = 4, F
+    // WINTER = 1, W
+    // SPRING = 2, S
+    // SUMMER = 3, M
+    const handleStartQuarterOnChange = (event) => {
+        setStartQuarter(event.target.value);
+    };
+
+    const handleEndQuarterOnChange = (event) => {
+        setEndQuarter(event.target.value);
+    };
+
+    const handleSubjectAreaOnChange = (event) => {
+        setSubjectArea(event.target.value);
+    };
+
+    const handleCourseNumberOnChange = (event) => {
+        setCourseNumber(event.target.value);
+    }
+
+    const handleCourseSufOnChange = (event) => {
+        setCourseSuf(event.target.value);
+    }
+    // Note: Not all possible courses were able to be added in the subject area list as many of them were dead links that provided no information
```

Comment:
Can you give a bit more detail about what this means, or provide an example? I'm confused as to what this comment means.